src/LiveComponent/doc/index.rst

```diff
@@ -3775,6 +3778,55 @@ uses Symfony's test client to render and make requests to your components::
     The ``InteractsWithLiveComponents`` trait can only be used in tests that extend
     ``Symfony\Bundle\FrameworkBundle\Test\KernelTestCase``.
 
+Test Live Collection Type
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Just a quick note here, in RST you need to use double backtiks to mark technical words (you could not know ;)  )

```diff
- the above `submitForm` helper) containing a `LiveCollectionType`, you
+ the above ``submitForm`` helper) containing a ``LiveCollectionType``, you
```